admin-core/assets/src/dashboard-app/pages/welcome/Welcome.js

```diff
@@ -2,93 +2,113 @@ import { __ } from '@wordpress/i18n';
 											onClick={ onCreateNewPageClick }
 										>
-											{__( 'Create New Page', 'ultimate-addons-for-gutenberg' )}
+											{ __( 'Create New Page', 'ultimate-addons-for-gutenberg' ) }
 										</button>
 										<button
 											type="button"
 											className="inline-flex items-center px-4 py-2 border border-slate-300 shadow-sm text-sm font-medium rounded-md text-slate-500 bg-white focus-visible:bg-slate-50 hover:bg-slate-50 focus:outline-none"
 											onClick={ onReadFullGuideClick }
 										>
-											{__( 'Visit Our Website', 'ultimate-addons-for-gutenberg' )}
+											{ __( 'Visit Our Website', 'ultimate-addons-for-gutenberg' ) }
 										</button>
 									</span>
 								</div>
 							</div>
 						</section>
 					</div>
 
-					{/* Right column */}
+					{ /* Right column */ }
 					<div className="space-y-4 flex h-full flex-col justify-between">
 						<section aria-labelledby="section-2-title">
 							<h2 className="sr-only" id="section-2-title">
 								Section title
 							</h2>
 							<div className="box-border rounded-md bg-white shadow-sm overflow-hidden transition hover:shadow-hover">
 								<div className="p-8 pr-10">
-									<h3 className='text-slate-800 text-xl font-medium pb-2'>
-										{ __(
-											'Knowledge Base',
-											'ultimate-addons-for-gutenberg'
-										) }
+									<h3 className="text-slate-800 text-xl font-medium pb-2">
+										{ __( 'Knowledge Base', 'ultimate-addons-for-gutenberg' ) }
 									</h3>
-									<p className='text-slate-500 text-sm pb-2 pr-2'>
+									<p className="text-slate-500 text-sm pb-2 pr-2">
 										{ __(
 											'Learn everything you need to know about the Spectra plugin with our comprehensive documentation.',
 											'ultimate-addons-for-gutenberg'
 										) }
 									</p>
-									<a className='text-base text-spectra focus:text-spectra focus-visible:text-spectra-hover active:text-spectra-hover hover:text-spectra-hover underline' href='https://wpspectra.com/docs/' target='_blank' rel="noreferrer">
-										{ __(
-											'Browse Now',
-											'ultimate-addons-for-gutenberg'
-										) }
+									<a
+										className="text-base text-spectra focus:text-spectra focus-visible:text-spectra-hover active:text-spectra-hover hover:text-spectra-hover underline"
+										href="https://wpspectra.com/docs/"
+										target="_blank"
+										rel="noreferrer"
+									>
+										{ __( 'Browse Now', 'ultimate-addons-for-gutenberg' ) }
+									</a>
+									<a
+										className="text-base text-spectra focus:text-spectra focus-visible:text-spectra-hover active:text-spectra-hover hover:text-spectra-hover no-underline"
+										href="https://wpspectra.com/docs/"
+										target="_blank"
+										rel="noreferrer"
+									>
+										{ ' ' }
+										→{ ' ' }
```

Comment:
Will this work as intended?
If so, imlpement in all instances.

```suggestion
										{ ' → ' }
```